.github/workflows/Tests.yaml

```diff
@@ -433,3 +446,86 @@ jobs:
           # We get occasional random timeouts, repeat tests to see if
           # it is a random timeout or systematic
           ctest -j2 --output-on-failure -L inputfiles --repeat after-timeout:3
+
+  # Release a new version on scheduled or manual events when the tests pass.
+  # Only enable this on the `sxs-collaboration/spectre` repository (not on
+  # forks).
+  release_version:
+    name: Release version
+    if: >
+      github.repository == 'sxs-collaboration/spectre' &&
+        (github.event_name == 'schedule'
+          || (github.event_name == 'workflow_dispatch'
+            && github.event.inputs.release_version != ''))
+    needs:
+      - check_files_and_formatting
+      - doc_check
+      - unit_tests
```

Comment:
Yes I was wondering about that as well. My suggestion here would be to require the tests pass, and if one fails randomly  we [rerun the workflow](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/). Better safe than sorry :)

Reply:
If I'm understanding correctly you're saying we just manually restart the tagging portion of CI? Or rerun the whole CI?

Reply:
I think GitHub only allows manually dispatching the whole workflow. So when the scheduled release fails randomly, we click "Dispatch Workflow", it will run the tests again and then the tagging, and probably not fail again (else we should really do something about the random test and not release a failing test).